### PR TITLE
feat: RN-1197: update Admin Panel table columns

### PR DESCRIPTION
**Pull request by jaskfla**
### Changes

- Hides a few columns in the Users, Permissions, Surveys and Survey Responses pages in Admin Panel. Hidden fields remain accessible in the “edit” menu.
- Replaces “Child ID” column with “Child code” in Map Overlay Group Relations page.

### CODE REVIEW

File: packages/admin-panel/src/pages/resources/MapOverlayGroupRelationsPage.jsx [hunk 1]
```diff
@@ -22,13 +22,13 @@ const FIELDS = [
...
     type: 'tooltip',
     editConfig: {
       optionsEndpoint: 'mapOverlays',
-      optionLabelKey: 'mapOverlay.id',
-      optionValueKey: 'mapOverlay.id',
+      optionLabelKey: 'mapOverlay.code',
+      optionValueKey: 'mapOverlay.code',
       canCreateNewOptions: true,
       sourceKey: 'child_id',
     },
```
Comment (jaskfla) [inline packages/admin-panel/src/pages/resources/MapOverlayGroupRelationsPage.jsx:34]: Less confident about this one. Works, but I don’t understand it from first principles, so please take an extra few seconds on this one 😅

Comment (alexd-bes) [reply packages/admin-panel/src/pages/resources/MapOverlayGroupRelationsPage.jsx:34]: I think this should work, assuming you have tested it?

Comment (jaskfla) [reply packages/admin-panel/src/pages/resources/MapOverlayGroupRelationsPage.jsx:34]: The autofill in the **New map overlay group relations* modal works, but I’ll double check it actually creates a valid entry in the Postgres table

Comment (jaskfla) [reply packages/admin-panel/src/pages/resources/MapOverlayGroupRelationsPage.jsx]: Turns out there are a few complications here. [Will make the updates to the **Map overlay group relations** page in a separate PR](https://linear.app/bes/issue/RN-1197/update-table-columns#comment-6f40b56e)

**Pull request merged**